articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -35,6 +37,10 @@ In either case, you are need to provide the fault domain count for your host gro
 
 You can also decide to use both availability zones and fault domains.
 
+Enabling ultra disks is a host group level setting and cannot be changed after a host groups os created.
```

Comment:
Need consistency with capitalization of ultra disk/Ultra disk. Per the ultra docs, I think both should be lower case

Reply:
Yeah, overlooked this, fixed it, thanks Brittany!